logging/performance_reliability/logging-input-spec-filtering.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="logging-input-spec-filtering"]
+= Filtering logs by metdata
+:context: logging-input-spec-filtering
+
+toc::[]
+
+
+You can filter logs in the `ClusterLogForwarder` CR to select or ignore the entire log stream based on metadata using the `input` selector. As an administrator, you can include or exclude log collection according to your interest. As a developer, you can include or exclude log collection to reduce the memory and CPU load on the collector. The following options are available for log filtering:
```

Comment:
The concerns and resolution are the same for devs and admins?  maybe combine the sentence?
"As an administrator or developer, you can include or exclude log collection to reduce the memory and CPU load on the collector."

Reply:
Made this change! Thanks.